apps/cde-ui/src/app/components/file-upload/file-upload.component.html

```diff
@@ -1,15 +1,26 @@
-<div class="column-content">
-  @if (!file && fileTitle()) {
-    <h3 class="subtitle">{{ fileTitle() }}</h3>
+<div
+  [class.layout-a]="layout() === 'layout-a'"
```

Comment:
Work with @LibbyUX  to remove inconsistent layouts between csv and color map upload areas